draft-packages/avatar/docs/Avatar.stories.tsx

```diff
@@ -40,121 +36,261 @@ DefaultStory.args = {
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser
+          size="large"
+          avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser
+          size="large"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser={false}
+          size="large"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials
+          isCurrentUser={false}
+          size="large"
+        />
+      </StoryWrapper.Row>
+      <StoryWrapper.Row rowTitle="Medium">
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser
+          size="medium"
+          avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser
+          size="medium"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser={false}
+          size="medium"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials
+          isCurrentUser={false}
+          size="medium"
+        />
+      </StoryWrapper.Row>
+      <StoryWrapper.Row rowTitle="Small">
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser
+          size="small"
+          avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser
+          size="small"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials={false}
+          isCurrentUser={false}
+          size="small"
+        />
+        <Avatar
+          fullName="Jane Doe"
+          disableInitials
+          isCurrentUser={false}
+          size="small"
+        />
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <div style={{ marginTop: "4rem" }}>
+      <StoryWrapper>
+        <StoryWrapper.RowHeader
+          headings={["Initals Unicode", "Initals Long", "Company Avatar"]}
+        />
+        <StoryWrapper.Row rowTitle="XX-Large">
+          <Avatar
+            fullName="李存信"
+            disableInitials={false}
+            isCurrentUser
+            size="xxlarge"
+          />
+          <Avatar
+            fullName="Spicy Jalapeno Bacon Ipsum Dolor Amet Aute Elit Chicken Mollit"
```

Comment:
```suggestion
            fullName="Spicy Jalapeno Taco Bacon Ipsum Pretzel Dolor Amet Nacho Elit Chicken"
```

I changed the string in #2555 because it was still having false positives before, but I battle tested the new one so it shouldn't happen again